cmd/describe/describe.go

```diff
@@ -17,27 +17,32 @@ limitations under the License.
 package describe
 
 import (
-	"fmt"
-
 	"github.com/spf13/cobra"
 )
 
 const (
-	volumeCommandHelpText = `# Show detail of a specific OpenEBS resource:
-$ kubectl openebs describe [volumes|storage|pvc] [name]
+	volumeCommandHelpText = `Show detailed description of a specific OpenEBS resource:
+
+Usage:
+  kubectl openebs describe [volume|storage|pvc] [...names] [flags]
 
-# Describe a Volume:
-$ kubectl openebs describe volume pvc-abcd -n [namespace] [options]
+Describe a Volume:
+  kubectl openebs describe volume [...names] [flags]
 
-# Describe PVCs present in the same namespace:
-$ kubectl openebs describe pvc [name1] [name2] ... [nameN] -n [namespace] [options]
+Describe PVCs present in the same namespace:
```

Comment:
```suggestion
Describe PVCs present in the default namespace:
```
Currently the current namespace is not picked up from the kube-context, `https://github.com/kubernetes/cli-runtime/issues/9`

Reply:
Did not understand this change. From the word same namespace, I meant that incase the user is describing multiple PVCs for ex, `kubectl openebs describe pvc mongo-pvc cstor-pvc -n cstor`, both the PVCs should belong to the `cstor` namespace. The user cannot describe PVCs in different namespace together using `-n` flag, as the `-n` flag accepts only one value.

Reply:
Oh, got your context too. I had missed the `s` in the `PVCs`